### PR TITLE
Test mock aggregator

### PR DESCRIPTION
**Pull request by shanejearley**
Account for PoR total Beacon validator balance in manager stake (`getStake()`). Hoping to quickly merge any completed incremental updates to through to the master copy to make sure Hacken is looking at latest (no code freeze yet).

### CODE REVIEW

**Comment by shanejearley** [comment]
One more commit coming in.

File: contracts/ethereum/src/CasimirPoR.sol [hunk 1]
```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
```
Comment (ccali11) [inline contracts/ethereum/src/CasimirPoR.sol:1]: The code in this file looks good overall, but I have a few comments:

- The getConsensusStake function is missing the override keyword, as it overrides the interface function getConsensusStake() from ICasimirPoR.
- In the getPoRAddressList function, the if (startIndex > endIndex) condition can be removed by changing the while loop to a for loop and setting the loop counter to startIndex directly, but up to you if you want to do that.

Overall, the code looks clean and well-structured, and these are minor suggestions that can improve its readability and maintainability.

**Review by ccali11 — APPROVED** [review]
Not ready to say I can fully assess the solidity code yet, but the code seems well-commented and well-structured. The contracts make use of interfaces and libraries to modularize the code and minimize duplication. Overall, the code appears to be written with good practices in mind. 👍

File: contracts/ethereum/README.md [hunk 2]
```diff
@@ -85,21 +94,25 @@ Core internal contracts and interfaces are located in the [src](./src) directory
...
-Development-only contracts and interfaces are located in the [src/dev](./src/dev) directory.
+| [DepositContract](./src/vendor/interfaces/IDepositContract.sol) | Accepts Beacon deposits | Todo |
+| [Functions](./src/vendor/Functions.sol) | Provides a library for Chainlink functions | Todo |
+| [FunctionsBillingRegistry](./src/vendor/interfaces/FunctionsBillingRegistryInterface.sol) | Manages Chainlink function billing | Todo |
+| [FunctionsClient](./src/vendor/FunctionsClient.sol) | Executes Chainlink function requests | Todo |
+| [FunctionsOracle](./src/vendor/interfaces/FunctionsOracleInterface.sol) | Handles Chainlink function requests | Todo |
+| [KeeperRegistry](./src/vendor/interfaces/IKeeperRegistry.sol) | Manages Chainlink upkeeps | Todo |
+| [SSVNetwork](./src/vendor/interfaces/ISSVNetwork.sol) | Connects distributed validators | Todo |
+| [SSVToken](./src/vendor/interfaces/ISSVToken.sol) | Serves as operator utility token | Todo |
+| [WETH](./src/vendor/interfaces/IWETH.sol) | Wraps ETH for swapping | Todo |
```
[the remark this answers is not in the document]
Comment (shanejearley) [reply contracts/ethereum/README.md:111]: Wrapped ETH is used to swap for LINK and SSV.

File: contracts/ethereum/src/CasimirPoR.sol (hunk 1 above)
Comment (shanejearley) [reply contracts/ethereum/src/CasimirPoR.sol:1]: Thanks, the getPoRAddressList from that line mentioned and below is pulled right from their example implementation, and they're the ones giving specs for the PoRAddressList interface that the Chainlink nodes consume so I'm going to keep it for now.

**Pull request merged**